.gitignore

```diff
@@ -98,3 +98,4 @@ portal/reactapp/*
 package.json
 .vscode/
 *.tramp_history
+db-scripts/src/main/resources/cgds-test.sql
```

Comment:
I'm not sure why this was added to .gitignore

Reply:
ah .. after reading through to the end, I guess it is because it *is* left in the source tree. Maybe by creating it in the target tree, this would not be necessary here.